pandas/io/tests/json/test_json_norm.py

```diff
@@ -133,6 +133,36 @@ def test_shallow_nested(self):
         expected = DataFrame(ex_data, columns=result.columns)
         tm.assert_frame_equal(result, expected)
 
+    def test_shallow_nested_with_separator(self):
```

Comment:
Also add test with the default separator, and ensure that the columns are `A.A, A.B`.